docs/docs/api/07-Configuration/askui-ui-control-client.md

```diff
@@ -44,6 +44,15 @@ ___
 The address of the askui Inference server.
 ___
 
+### resize
+
+- **resize**: `number?` - Default: `undefined`
+  The Side length of the target image to resize to in px. Your screenshot image will be resized with the original aspect ratio, and the lengths image side will be equal to this number. This can be used to reduce the inference time by reducing the request size in case of a bad internet connection. But it can cause a decrease in the prediction quality.
```

Comment:
```suggestion
  The side length of the target image to resize to in px. Your screenshot image will be resized with the original aspect ratio, and the lengths image side will be equal to this number. This can be used to reduce the inference time by reducing the request size in case of a bad internet connection. But it can cause a decrease in the prediction quality.
```